File: Backend/src/services/interfaces/IEmployeesServices.ts
import { Employees } from "../../domain/Entities/Employees";
import { IServices } from "./IServices";

interface IEmployeesServices extends IServices<Employees> {
    GetByEmail(email: string): Promise<Employees>;
    VerifyEmailExists(email:string): Promise<boolean>
}
export { IEmployeesServices }

File: BackEnd/Services/Services/IServices/IEmployeesServices.cs
using Domain.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bussines.Services.IServices
{
    public interface IEmployeesServices : IServices<Employees>
    {
        Task<Employees> GetByCode(string code);
    }
}
